nixos/tests/nextcloud/with-postgresql-and-redis.nix

```diff
@@ -46,7 +46,7 @@ in {
       systemd.services.redis = {
         preStart = ''
           mkdir -p /var/run/redis
-          chown ${config.services.redis.user}:${config.services.nginx.group} /var/run/redis
+          chown redis:${config.services.nginx.group} /var/run/redis
         '';
         serviceConfig.PermissionsStartOnly = true;
```

Comment:
this is deprecated and should be removed.

Reply:
removed